### PR TITLE
Adding FIS global

### PR DESCRIPTION
**Pull request by ashishdawale20**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [ ] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [ ] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [ ] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [ ] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [ ] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [ ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Review by dougaitken — CHANGES_REQUESTED** [review]
@ashishdawale20 A few changes:

- [ ] Company name would match in the readme and the profile
- [ ] global should be capitalised
- [ ] File name should be all lowercase, but keep he `-` for the gap
- [ ] Match the regions, is it USA only as in readme or also Asia as mentioned in the profile.

**Comment by ashishdawale20** [comment]
> @ashishdawale20 A few changes:
> 
> * [ ]  Company name would match in the readme and the profile
> * [ ]  global should be capitalised
> * [ ]  File name should be all lowercase, but keep he `-` for the gap
> * [ ]  Match the regions, is it USA only as in readme or also Asia as mentioned in the profile.

@dougaitken dougaitken Thanks for the feedback - i have correct as per suggestion, can you take a look once again?

File: README.md [hunk 1]
```diff
@@ -264,6 +264,7 @@ Name | Website | Region
 [FingerprintJS](/company-profiles/fingerprintjs.md) | https://fingerprintjs.com | Worldwide
 [Fire Engine Red](/company-profiles/fire-engine-red.md) | https://fire-engine-red.com/ | Worldwide
 [Fireball Labs](/company-profiles/fireball-labs.md) | https://www.fireballlabs.com | Germany
+[FIS GLOBAL](/company-profiles/FIS-global.md) | https://www.fisglobal.com/ | USA, UK, Asia
```
Comment (dougaitken) [inline README.md:267]: Me the file reference lowercase here @ashishdawale20

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashishdawale20) [reply README.md]: My bad, corrected it now.

**Review by dougaitken — APPROVED** [review]

**Pull request merged**